src/Orleans/Runtime/GrainReference.cs

```diff
@@ -20,6 +21,8 @@ public class GrainReference : IAddressable, IEquatable<GrainReference>, ISeriali
         [NonSerialized]
         private static readonly TraceLogger logger = TraceLogger.GetLogger("GrainReference", TraceLogger.LoggerType.Runtime);
 
+        private static readonly CancellationTokenManager cancellationTokenManager = new CancellationTokenManager();
```

Comment:
The only need in `CancellationTokenManager ` was to remove unrelated by responsibility code from the class that using it. Removed.